libs/ptg-ui-web-components/src/components/ptg-button/ptg-button.tsx

```diff
@@ -7,13 +7,16 @@ import { HTMLStencilElement } from '@stencil/core/internal';
   shadow: true,
 })
 export class PtgButton {
-  @Prop() text: string;
+  @Prop() text?: string= "";
   @Prop() appearance: string;
   @Prop() btnIconAlignment?: string = 'right';
   @Element() hostElement: HTMLStencilElement;
   hasbtnIconSlot: boolean;
 
   componentWillLoad() {
+    if(this.text === ""){
```

Comment:
this.text === ""  instead use !this.text

Reply:
Feedback issue resolved